# see available mappings on src/maqui_joy/xbox.py

# pause button
b_pause: 'START'

# movement axes
a_angular: 'RS_HORZ'
a_linear_x:  'LS_VERT'
a_linear_y:  'LS_HORZ'

# cmd velocities
max_linear_vel: 0.35  # caution: robot max vel is 0.35.
max_angular_vel: 0.5  # caution: robot max vel is 1.0. However, the response is really weird for values higher than 0.6
